### DISCUSSION TITLE
Crawlers for getting all the URLs associated with a domain

### DISCUSSION THREAD

**Discussion by devu-62442** [projectdiscovery/nuclei#3401] [Q&A] [answered]
Do we have any flag to get all the URLs associated with a domain? 
The recursive scans to all the URLs can help in getting more information in the domains and security issues.

**Comment by ehsandeep** [comment, marked as answer]
@devu-62442 as for now you can do `cat urls.txt | katana | nuclei` to achieve the same, later [katana](https://github.com/projectdiscovery/katana) will be integrated into nuclei to do this natively.

> **Note**: make sure to run relevant templates only on all URLs as most of the nuclei templates are written to run on root URLs.

**Reply by devu-62442** [reply]
Thank You so much. Appreciate it.
